Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -1374,8 +1374,9 @@ final class PackageToolTests: CommandsTestCase {
             let packageDir = tmpPath.appending(components: "MyPackage")
             try localFileSystem.writeFileContents(packageDir.appending(components: "Package.swift")) {
                 $0 <<< """
-                // swift-tools-version: 999.0
+                // swift-tools-version: 5.6
```

Comment:
This is an incidental fix that will be needed once we mark the 5.6 branch as no longer a development branch (so it stops accepting 999).  The PR that added this was still open when all the plugins were changed from 999 to 5.6 after evolution proposal acceptance.